Atomic/images.py

```diff
@@ -259,7 +259,8 @@ def _filter_include_image(self, image_obj):
                                  "Please choose from {}".format(var, filterables))
             if var == "type":
                 var = "str_backend"
-            if getattr(image_obj, var, None).lower() != value.lower():
+
+            if value.lower() not in str(getattr(image_obj, var, None)).lower():
```

Comment:
since we are touching this line, I wonder about the missing case here.  Do we really want to compare the value to the string "none"?

Should we do something like this instead?

`hasattr(image_obj, var) and value.lower() in image_obj.var.lower()`